.github/workflows/bat.yml

```diff
@@ -170,3 +170,8 @@ jobs:
           grep "deploying" buildlog.txt
           ! grep "checking" buildlog.txt
           rm buildlog.txt
+
+      - name: Verify environment variable is setup for MATLAB plugin override
+        run: |
+          set | grep "MW_MATLAB_BUILDTOOL_DEFAULT_PLUGINS_FCN_OVERRIDE" | grep "matlab.ciplugins.github.getDefaultPlugins"
```

Comment:
Should this environment variable actually stick around? To reduce side effects of running `run-build`, we should consider cleaning this up.

Reply:
+1 also feels like we're testing implementation details instead of the actual behavior here

Reply:
This is one of the process which lets us know that we are using our custom plugins. Other way is to check the log however reading workflow log is to straight forward in Github I guess I tried with `gh` command however that did not help me grep it. Seems like we need to download log through curl API . which did not seem to work.  As I have mentioned in the PR details I am working on to see how I could grep the workflow logs to check the actual functionality. 
let me know if any suggestions.

Reply:
Done